packages/ts-endpoint-express/src/index.ts

```diff
@@ -77,15 +78,18 @@ export const AddEndpoint: AddEndpoint = (router, ...m) => (e, controller) => {
 
     const taskRunner = pipe(
       args,
-      E.mapLeft(
-        (errors) =>
-          new IOError(DecodeErrorStatus, 'error decoding args', { kind: 'DecodingError', errors })
-      ),
+      E.mapLeft((errors) => {
+        console.log('errors', PathReporter.report(E.left(errors)));
```

Comment:
I guess this log is intentional?

Reply:
P.S. the new `io-ts` codecs have a much clearer way of reporting errors, do you think we could switch to that in the foreseeable future?

Reply:
> I guess this log is intentional?

Yep, my bad! Removed!

> P.S. the new io-ts codecs have a much clearer way of reporting errors, do you think we could switch to that in the foreseeable future?

I didn't take a look at that feature, but does it mean we need to change all the `io-ts` definitions to be compatible?

Reply:
I guess so, but I just briefly took a look at it.. Maybe it can be backported